### ISSUE TITLE
Peer Review

### ISSUE THREAD

**Issue by AndresPitta** [UBC-MDS/pysketball#63]
Hi,

Release [2.2](https://github.com/UBC-MDS/pysketball/tree/2.2) addresess the feedback given in the following issues:

### **Issues addressed by @AndresPitta**:
Issue [#60 ](https://github.com/UBC-MDS/pysketball/pull/60#issue-392129884)

This commit contains the following changes:

- The README.md file was modified to contain information about the dependecies and how to install the correct version of them. This addresses @kvarada's following feedback on issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-602124775):

here
>Installation directions might benefit from some clarification. I had to do the following in order to get it working.
>
>`conda install selenium`
`pip install python-semantic-release`

here

> Good that you include the instruction to put the chromedriver path in the system path variable. I guess you do not mean PATH_TO_CHROMEDRIVER.EXE in the following command for Linux and MacOS.
> 
> `export PATH="<PATH_TO_CHROMEDRIVER.EXE>:$PATH"`

and here
>Tests
>When I ran your tests, two tests failed with the following error.
`AttributeError: module 'altair.vegalite' has no attribute 'v4'`

- The README.md file was also modified to have the correct example for the function `nba_ranking`, in order to address @kvarada and @zoepan00 feedback on  issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-602124775):

here:

> Step 3 example nba_ranking(nba_2018_regular, 'PLAYER' , 'GP', top = 2, ascending = False, fun = 'mean') is not working

and here:

> In your README, in the following usage example of the function nba_ranking, the 'PLAYER' column is undefined in the scraped CSV and so it gives an error. Replacing it with 'NAME' worked for me.
> nba_ranking(nba_2018_regular, 'PLAYER' , 'GP', top = 2, ascending = False, fun = 'mean')

- `nba_ranking`'s error message was updated to address @zoepan00's feedback on issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-601541684): 

> nba_ranking: Error message for fun argument is not updated. Currently it says The fun argument should be either var or mean

### **Issues addressed by @vanandsh**:
Issue [#59 ](https://github.com/UBC-MDS/pysketball/pull/59#issuecomment-602363715)

- Updated metadata for authors etc. information in pyproject.toml, given by @zoepan00 in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-601541684):

> I seem can't find the metadata for authors etc. information

- Updated docstring for season_type argument in nba_scraper.py, given by @zoepan00 in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-601541684): 

> Docstring for season_type argument is not updated. I think it would be either regular or postseason

- Updated docstring to include example outputs in nba_team_stats.py, to address @zoepan00:

> it would be nice to have the rendered outputs for nba_ranking, nba_boxplot, nba_team_stats 

and @kvarada ([#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-602124775)):

> Some of the usage examples in the docstrings are not complete. In many cases, the output is missing. For example, in your docstring of nba_ranking function, I would give an example with a toy CSV which has similar to your scraped CSV (e.g., NAME, GP columns). Also, showing the output of the example (the dataframe and not the plot) would help.

- Updated code to round the numbers for mean and std columns in nba_team_stats.py, given by @zoepan00:

> It would be nice if you can round the numbers for mean and std columns.

### **Issues addressed by @carlinakim**:
Issue [#58 ](https://github.com/UBC-MDS/pysketball/pull/58#issuecomment-602366359)

- Changed `test_nba_ranking` to not depend on V4 objects, addressing @kvarada's feedback in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-602124775):

> Tests
> When I ran your tests, two tests failed with the following error.
> AttributeError: module 'altair.vegalite' has no attribute 'v4'

- Changed `nba_boxplot` to address @zoepan00's feedback in issue [#31](https://github.com/UBC-MDS/software-review/issues/31#issuecomment-601541684):

> Error message helps for the case if both position and teams have values. I think it would be nice to also include an error message if I input other columns names to the position argument(for example an error message for position="TEAM"), or change the position argument type to boolean.
